### PR TITLE
[MBL-17005][Student] Offline content screen interaction tests

### PR DESCRIPTION
**Pull request by kristofnemere**
Test plan: -

refs: MBL-17005
affects: Student
release note: none

### CODE REVIEW

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Review by tamaskozmer — APPROVED** [review]

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

File: apps/student/src/androidTest/java/com/instructure/student/ui/interaction/OfflineContentInteractionTest.kt [hunk 1]
```diff
@@ -0,0 +1,377 @@
...
+    @Test
+    @TestMetaData(Priority.IMPORTANT, FeatureCategory.OFFLINE_CONTENT, TestCategory.INTERACTION, false)
+    fun selectCourse() {
+        val data = createMockCanvas()
+        goToOfflineContent(data)
+        val course = data.courses.values.first()
+        manageOfflineContentPage.expandCollapseItem(course.name)
+        manageOfflineContentPage.assertCheckedStateOfItem(course.name, MaterialCheckBox.STATE_UNCHECKED)
+        manageOfflineContentPage.changeItemSelectionState(course.name)
+        getCourseItemNames(data, course).forEach { manageOfflineContentPage.assertCheckedStateOfItem(it, MaterialCheckBox.STATE_CHECKED) }
```
[the remark this answers is not in the document]
Comment (kristofnemere) [reply apps/student/src/androidTest/java/com/instructure/student/ui/interaction/OfflineContentInteractionTest.kt:113]: that's included, the getCourseItemNames method returns the course name also

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Pull request merged**